mava/adders/base.py

```diff
@@ -36,7 +36,7 @@ class ParallelAdder(adders.Adder):
     ```python
     # Reset the environment and add the first observation.
     timestep = env.reset()
-    adder.add_first(timestep.observation)
```

Comment:
Why is this edited?

Reply:
No idea :raised_eyebrow: Please change it back if you get the chance.